Package@swift-5.7.swift

```diff
@@ -0,0 +1,31 @@
+// swift-tools-version:5.8
+import PackageDescription
+
+let package = Package(
+  name: "ResilientDecoding",
+  platforms: [
+      .iOS(.v12),
+      .tvOS(.v12),
+      .watchOS(.v5),
+      .macOS(.v10_14),
+  ],
+  products: [
+    .library(
+      name: "ResilientDecoding",
+      targets: ["ResilientDecoding"]),
+  ],
+  targets: [
+    .target(
+      name: "ResilientDecoding",
+      dependencies: []),
+    .testTarget(
+      name: "ResilientDecodingTests",
+      dependencies: ["ResilientDecoding"]),
+  ]
+)
+
+for target in package.targets {
+  var settings = target.swiftSettings ?? []
+  settings.append(.enableExperimentalFeature("StrictConcurrency"))
+  target.swiftSettings = settings
+}
```

Comment:
If you're intending this file to be swift tools version 5.8, I'd recommend writing this as:
```suggestion
    .target(
      name: "ResilientDecoding",
      dependencies: [],
      swiftSettings: [
        .enableUpcomingFeature("StrictConcurrency")
      ]),
    .testTarget(
      name: "ResilientDecodingTests",
      dependencies: ["ResilientDecoding"],
      swiftSettings: [
        .enableUpcomingFeature("StrictConcurrency")
      ])
  ]
)
```

Note that this is an _upcoming_ rather than _experimental_ feature. In tools version 5.7, you indeed do need to call this experimental IIRC.

Reply:
With 5.8, this is [still considered an experimental feature](https://github.com/apple/swift/pull/66991#issue-1779561033), so we need to keep it as `. enableExperimentalFeature`. I will inline it to the targets, however, as you suggest.

Reply:
Welp I may have some code in my own projects to update then